test/e2e/apimachinery/aggregator.go

```diff
@@ -94,7 +94,7 @@ var _ = SIGDescribe("Aggregator", func() {
 		    Description: Ensure that the sample-apiserver code from 1.17 and compiled against 1.17
 			will work on the current Aggregator/API-Server.
 	*/
-	framework.ConformanceIt("Should be able to support the 1.17 Sample API Server using the current Aggregator", func() {
+	framework.ConformanceIt("Privileged", "Should be able to support the 1.17 Sample API Server using the current Aggregator", func() {
```

Comment:
there is opportunity to have these as constants under /test/e2e/profiles:
```
import "k8s.io/kubernetes/test/e2e/profiles"

// usage:
// profiles.ConformanceBase
// profiles.ConformancePriviliged
```

ideally, the framework should be unaware of the profile declarations.

Reply:
Yeah, i'm not even sure framework.ConformanceIt is really appropriate.  I think the package being called conformance`conformance.BaseProfileIt(...)` in general would be clearer.

Conformance is a subset of our suite.

Reply:
that sounds reasonable

Reply:
I'm fine with all of these as improvements but don't view them as necessary to land in order to land segmenting tests into profiles via this PR.

If we're going to shuffle packages around and rename, I would suggest waiting to do that as part of testing-commons' effort to detangle test/e2e/framework enough to move into staging.